templates/how-it-works.html

```diff
@@ -82,47 +82,44 @@ <h2>Physical availability zones</h2>
     </div>
 </section>
 
-<section class="p-strip is-deep">
+<section class="p-strip is-deep u-no-padding--bottom">
     <div class="row">
         <div class="col-8">
             <h2>The node lifecycle</h2>
             <p>Each machine (&ldquo;node&rdquo;) managed by MAAS goes through a lifecycle &mdash; from its enlistment or onboarding to MAAS, through commissioning when we inventory and can setup firmware or other hardware-specific elements, then allocation to a user and deployment, and finally they are released back to the pool or retired altogether.</p>
         </div>
     </div>
+</section>
+<div class="u-sv3"></div>
```

Comment:
Why do we need to make more space between strips? The strip spacing should be even.

Reply:
Ok thanks